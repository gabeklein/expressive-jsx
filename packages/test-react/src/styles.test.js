const test = require("./_adapter");

test("basic style", `
  () => {
    color: green;

    <this>Hello World</this>
  };
`);