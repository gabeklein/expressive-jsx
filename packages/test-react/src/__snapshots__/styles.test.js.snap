// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`basic style 1`] = `
"() => create(\\"div\\", {
  \\"className\\": \\"element_26b\\"
}, \\"Hello World\\");

CSS.put(\`
	/* 1 */
	.element_26b { color: green }
\`);"
`;

exports[`basic style 2`] = `
"() => <div className=\\"element_26b\\">Hello World</div>;

CSS.put(\`
	/* 1 */
	.element_26b { color: green }
\`);"
`;
